analyzer.go

```diff
@@ -104,18 +104,23 @@ func (a *Analyzer) NotifyReviewEvent(ctx context.Context, e *pb.ReviewEvent) (
 			continue
 		}
 
-		// analyze only changes in Golang
-		if strings.HasPrefix(strings.ToLower(change.Head.Language), "go") {
-			tryToSaveTo(change.Head, tmp)
+		file := change.Head
+		if tryToSaveTo(file, tmp) != nil {
```

Comment:
you have 2 problems in this code:
1. most probably you want to increase `saved` when `tryToSaveTo(file, tmp)` does not return an error. Currently it works opposite.
2. Here `log.Errorf(err, "failed to write file %q", file.Path)` you actually log `nil` error from the line 93: `changes.Recv()`. Most probably it's not what you intended to do.

Reply:
😑 you're right!